modules/ROOT/pages/type-definitions/directives/cypher.adoc

```diff
@@ -154,6 +154,26 @@ type Query {
 
 The downside of the latter approach is that you need to adjust the return object as you change your object type definition.
 
+== Input arguments
+The `@cypher` statement can access the query parameters by prepending `$` to the parameter name. For example:
+
+[source, graphql, indent=0]
+----
+type Query {
+    name(value: String): String @cypher(statement: "RETURN $value AS res", columnName: "res")
+}
+----
+
+The following GraphQL query will return the parameter  `value`:
+
+[source, graphql, indent=0]
+----
+query {
+  name(value: "aca")
```

Comment:
what's "aca"?

Reply:
random text will change for something better